app/models/event_log.rb

```diff
@@ -76,6 +77,10 @@ def event
     entry.description
   end
 
+  def requires_admin?
+    ACCESS_LIMITED_EVENTS.include? entry
```

Comment:
Do you think there's any value in adding a test for this method?

Reply:
I don't necessarily think this is necessary given it only refers to the content of a constant on line 58